### PR TITLE
Add the option to upload a photo using a url to it

### PR DESCRIPTION
**Pull request by octopusinvitro**
This PR adds a second form to add a photo from a url

## Screenshots

form

## Related issues

Part 4 of https://github.com/DemocracyClub/yournextrepresentative/issues/150

Partial fix for #150

### CODE REVIEW

File: moderation_queue/forms.py [hunk 1]
```diff
@@ -33,11 +53,27 @@ def clean(self):
...
+                        "justification for why we can use it.")
             raise ValidationError(message)
         return cleaned_data
 
 
+class UploadPersonPhotoUrlForm(forms.Form):
+    image_url = StrippedCharField(
+        widget=forms.URLInput(),
+    )
+    why_allowed_url = forms.ChoiceField(
```
Comment (octopusinvitro) [inline moderation_queue/forms.py]: Mmm... but the HTML will be generated with an `id` attribute that is called `why_allowed` non?
Which means we will have two elements in the same page with the same `id` attribute :thinking:

Comment (mhl) [inline moderation_queue/forms.py]: Oh yeah, good point!

File: moderation_queue/templates/moderation_queue/_photo-upload-url-form.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{% load i18n %}
+
+<h2>{% trans 'Upload a photo from a url' %}</h2>
```
Comment (hugovk) [inline moderation_queue/templates/moderation_queue/_photo-upload-url-form.html]: -> "Upload a photo from a URL"

File: moderation_queue/templates/moderation_queue/_photo-upload-url-form.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+  {{ url_form.non_field_errors }}
+  {% csrf_token %}
+
+  <p>
+    {% if queued_images %}{% blocktrans trimmed with name=person.name %}
+    If you still want to upload another photo of {{ name }}, you can also paste a url to the image (maximum size: 50 MB):
+    {% endblocktrans %}
+    {% else %}
+    {% trans "You can also paste a url to the image (maximum size: 50 MB):" %}
```
Comment (hugovk) [inline moderation_queue/templates/moderation_queue/_photo-upload-url-form.html]: -> "You can also paste a URL to the image"

File: moderation_queue/templates/moderation_queue/_photo-upload-url-form.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+<h2>{% trans 'Upload a photo from a url' %}</h2>
+
+<form id="person-upload-photo-url" action="{% url 'photo-upload-url' person_id %}" method="post" enctype="multipart/form-data">
+
+  {{ url_form.non_field_errors }}
+  {% csrf_token %}
+
+  <p>
+    {% if queued_images %}{% blocktrans trimmed with name=person.name %}
+    If you still want to upload another photo of {{ name }}, you can also paste a url to the image (maximum size: 50 MB):
```
Comment (hugovk) [inline moderation_queue/templates/moderation_queue/_photo-upload-url-form.html]: -> "you can also paste a url"

**Pull request merged**